examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
Landing on the same pod wasn’t the problem for me. With the PVC in Minikibe the two replicas are contending for ownership since it is not a storage class that allows of multiple attachments Via the ReadWriteOnce